### ISSUE TITLE
Invalid link to purchase on spigot

### ISSUE THREAD

**Issue by sorayasoon** [ds58/Panilla#167]
Hello, i want to purchase panilla but the link of paypal doesnt work... 
I'm contacting here cause I found no other way

**Comment by Incluuu** [comment]
![image](https://github.com/user-attachments/assets/aa5b53d7-450a-42fd-873e-da6fa9250705)
why, panilla on spigotmc is inactive and unavailable for new purchases?

**Comment by sorayasoon** [comment]
yes, when i click to ''buy now'', paypal says the link is invalid

**Comment by sorayasoon** [comment]
I think the main problem is the invalid paypal link, doesnt work, cause I can click on ''buy now'', but as the link is invalid, I cant buy

**Comment by masondaniels** [comment]
We've contacted Spigot about this issue. Shortly, I will be the maintainer of the plugin. I will be a bit busy this week, but plan to update and fix issues.
